onprc_ehr/src/org/labkey/onprc_ehr/issues/RestrictedIssueProviderImpl.java

```diff
@@ -0,0 +1,136 @@
+package org.labkey.onprc_ehr.issues;
+
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+import org.labkey.api.data.Container;
+import org.labkey.api.data.ContainerManager;
+import org.labkey.api.data.PropertyManager;
+import org.labkey.api.issues.Issue;
+import org.labkey.api.issues.RestrictedIssueProvider;
+import org.labkey.api.query.SimpleValidationError;
+import org.labkey.api.query.ValidationError;
+import org.labkey.api.security.Group;
+import org.labkey.api.security.SecurityManager;
+import org.labkey.api.security.User;
+import org.labkey.api.security.ValidEmail;
+import org.labkey.api.util.Pair;
+
+import java.util.List;
+import java.util.Objects;
+
+public class RestrictedIssueProviderImpl implements RestrictedIssueProvider
+{
+    private static final String CAT_RESTRICTED_ISSUE_DEF_PROPERTIES = "RestrictedIssueDefProperties-";
+    private static final String PROP_RESTRICTED_ISSUE_LIST = "issueRestrictedIssueList";
+    private static final String PROP_RESTRICTED_ISSUE_LIST_GROUP = "issueRestrictedIssueListGroup";
+
+    @Override
+    public void setRestrictedIssueTracker(Container c, String issueDefName, Boolean isRestricted)
+    {
+        setPropertyValue(c, issueDefName, PROP_RESTRICTED_ISSUE_LIST, String.valueOf(isRestricted));
+    }
+
+    @Override
+    public boolean isRestrictedIssueTracker(Container c, String issueDefName)
+    {
+        String value = getPropertyValue(c, issueDefName, PROP_RESTRICTED_ISSUE_LIST);
+        if (value != null)
+        {
+            return Boolean.parseBoolean(value);
+        }
+        return false;
+    }
+
+    @Override
+    public void setRestrictedIssueListGroup(Container c, String issueDefName, @Nullable Group group)
+    {
+        setPropertyValue(c, issueDefName, PROP_RESTRICTED_ISSUE_LIST_GROUP, null != group ? String.valueOf(group.getUserId()) : "0");
```

Comment:
the "0" here is for the null case right? Is there a reason that we couldn't just use `null` for the property value?